reference/tax.v3.yml

```diff
@@ -91,7 +91,12 @@ paths:
         '422':
           description: 'Unprocessable Entity, will include a specific error message referencing the issue.'
       description: |-
-        Set the HTTP Basic Authentication credentials for the specified provider. The configured `username` and `password` will be used to authenticate each API request to the Tax Provider from the associated store.
+        Set authentication information associated with a merchant's account on the tax provider's infrastructure:
+        - [HTTP Basic Authentication](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication) (developer.mozilla.org) credentials
+        - (optional) Tax provider profile used in customized endpoint urls for tax provider calls. This is available for only supporting tax providers.
```

Comment:
`This is available for only supporting tax providers.` This sentence reads strangely to me.
perhaps
`This is only available for tax providers that support this feature.`?

Reply:
Thanks for the feedback ✔️ , I changed it